commonlib/src/main/java/com/crisolutions/commonlib/utils/permissions/PermissionType.kt

```diff
@@ -0,0 +1,10 @@
+package com.crisolutions.commonlib.utils.permissions
+
+import android.Manifest
+
+enum class PermissionType(
```

Comment:
Although this is good to merge in, Im wondering if there is a different way to do this that wouldn't require updating the common library each time to add a new permission.